tedana/stats.py

```diff
@@ -74,10 +74,15 @@ def computefeats2(data, mmix, mask=None, normalize=True):
     # demean masked data
     if mask is not None:
         data = data[mask, ...]
+    # normalize data (minus mean and divide by std) in the last dimension
+    # so that least-squares estimates represent correlation values (data_R)
+    # assuming mmix are also normalized
     data_vn = stats.zscore(data, axis=-1)
 
     # get betas of `data`~`mmix` and limit to range [-0.999, 0.999]
     data_R = get_coeffs(data_vn, mmix, mask=None)
+    # Avoid abs(data_R) = 1 or -1. 
```

Comment:
Because the standardized parameter estimates from a multiple regression are not the same as correlation coefficients and often go above or below what is possible with correlation coefficients, this step also avoids values > 1 or < -1.